config/zones.json

```diff
@@ -2864,8 +2864,7 @@
       "productionPerModeForecast": "ENTSOE.fetch_wind_solar_forecasts",
       "price": "ENTSOE.fetch_price",
       "production": "ENTSOE.fetch_production"
-    },
-    "timezone": "Europe/Rome"
```

Comment:
This was a duplicate - I thought I'd tidy it as I went - https://github.com/tmrowco/electricitymap-contrib/pull/2851/files#diff-5fdb410dda63034a171c7bacd44c070de0c6a2d0ce5c0d5436b3626877b6a583R2837